pyproject.toml

```diff
@@ -176,54 +178,90 @@ ignore = [
 
 [tool.pixi.project]
 channels = ["conda-forge"]
-platforms = ["linux-64", "osx-arm64", "osx-64"]
+platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
```

Comment:
I am surprised this works. Shouldn't fail as `jaxlib` is not available on Windows?

Reply:
I pushed windows by mistake, it was an old attempt :)